docs/index.asciidoc

```diff
@@ -15,70 +15,99 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 [id="plugins-{type}s-{plugin}"]
 
-=== Example filter plugin
+=== PILAR Filter Plugin
 
 include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-Add plugin description here
+The PILAR plugin ports the https://dl.acm.org/doi/abs/10.1109/ICSE48619.2023.00077[PILAR parser] implementation for dynamic parsing of log events. The PILAR parser builds an n-gram dictionary of tokens in order to determine which tokens in a log event are considered dynamic and which are considered static. This enables the parser to dynamically extract out portions of the log without requring formal regular expressions.
+
+This plugin format performs especially well against logs that have a consistent format (eg. syslogs, Apache, MySQL) but unlike regex-based parsers like Grok and Dissect, this plugin can "learn" what parts of a log can be extracted out based on past log data that the plugin has analyzed.
```

Comment:
Why is learn in quotation marks here?

Reply:
Because it's not really learning anything, it's just using probability. I can reword